content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/dipole_sources/index.rst

```diff
@@ -7,43 +7,61 @@ Maxwell's Equations for Dipole Sources
     :file: ../../../../../underconstruction.html
 
 
-.. topic:: Purpose
+.. Purpose::
```

Comment:
I plan on doing a thorough look through everything before I suggest pulling it into master. That will be a little while though. Lots to do.